api/settings/urls.py

```diff
@@ -6,4 +6,5 @@
 urlpatterns = [
     path('api/auth/', include('rest_framework.urls', namespace='rest_framework')),
```

Comment:
what is api/auth? do you have another auth?

Reply:
ye its database access